hydromt_wflow/wflow_sediment.py

```diff
@@ -62,9 +62,9 @@ def setup_rivers(
         min_rivwth: float = 30,
         smooth_len: float = 5e3,
         output_names: Dict = {
-            "river_location__mask": "wflow_river",
-            "river__length": "wflow_riverlength",
-            "river__width": "wflow_riverwidth",
+            "river_location__mask": "river_mask",
+            "river__length": "river_mask_length",
+            "river__width": "river_mask_width",
```

Comment:
I did not check if you did that for wflow.py (can you?) but changes for these variables were made for the river part by Tjalling. They should be (but you can better revert as Tjalling made the changes)

```suggestion
            "river__length": "river_length",
            "river__width": "river_width",
```